XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -834,6 +834,10 @@ Some benefits of this design approach include that is (1) reduces the footprint
 #### A.1.14. Why doesn’t an `MPTokenIssuanceID` simply hash an issuer address and currency code?
 Primarily because we did not want MPT meta-data (e.g., currency code, asset precision, common name, currency symbol, etc.) to be part of an MPTs unique identifier. This mirrors the design of other tokenization primitives on other blockchain networks that have gained massive adoption, offering strong “prior art” (e.g., ERC-20 and ERC-721 tokens). For a more detailed discussion, see [here](https://github.com/XRPLF/XRPL-Standards/discussions/128).
 
+#### A.1.15. Why does `MPTokenIssuance` include an `AssetCode` instead of letting this exist solely in `MPTokenMetadata`?
+
+This specification generally prefers to have any MPT metadata reside inside of an issuer-defined `MPTokenMetadata` blob. However, `AssetScale` is an exception so that on-ledger transactors or other on-ledger components can properly perform mathematical computations on an MPT amount.
+
```

Comment:
I don't believe the scale is involved in computations in transactors,, @gregtatcam  could you confirm?

But having an individual AssetScale field is really useful for things like the explorer, where it can display the scaled amount by dividing `amount/10^scale`. If the scale was stored in the metadata, there is really no good way to get the scale

Reply:
> I don't believe the scale is involved in computations in transactors

Agreed, it's not used today as far as I can tell. However, I'm more curious about whether it will be used in the future (e.g., in the DEX or AMM?)

Reply:
I don't believe DEX or AMM were planning to use AssetScale either

iirc, the main rationale of having `AssetScale` to be a first class field was so web3 apps would have a standardized way to know what the scale is, where metadata was more free-form and hard to require

Reply:
> However, it could be argued that this field could be used on-chain in the future

Yeah, this is one key aspect I'm very interested in identifying. I don't think there are any current scenarios I can think of for "on-ledger" usage of this field, so from that perspective the wording in my PR is technically incorrect. 

However, it's possible that some future transactor (or even some future smart contract layer) might need this field, so from that perspective I think what I have in this PR should stay (but be appended -- see below).

> iirc, the main rationale of having AssetScale to be a first class field was so web3 apps would have a standardized way to know what the scale is, where metadata was more free-form and hard to require

Yes, I recall that rationale as well. In particular, I do worry about proper display and computation of MPT amounts by off-ledger applications, so how about this as an alternative for this PR:

```
This specification generally prefers to have any MPT metadata reside inside of an issuer-defined
 `MPTokenMetadata` blob. For that to work, we'll likely need to define a separate "MPT Metadata" JSON
 standard that could define a JSON payload to be embedded in `MPTokenMetadata` (sort of like [this one]
(https://eips.ethereum.org/EIPS/eip-1155#erc-1155-metadata-uri-json-schema)).

However, `AssetScale` is an exception to this guideline for three important reasons. 

1. **Future On-ledger Calculations**: While no existing on-ledger functionality (e.g., Payments, 
DEX, AMM, etc) requires `AssetScale`, it's possible that future on-ledger transactors or even future 
smart-contract layers may require this field in order to properly perform mathematical computations 
on an MPT amount.

3. **Off-ledger Amount Display**: It will be important for off-ledger applications (e.g., [XPRL Ledger Explorer]
(https://livenet.xrpl.org/)) to properly display MPT amounts in an appropriate form. For example,
 imagine a USD stablecoin issued as an MPT with a scale of `2`. In this case, a payment might 
be made for 100 units, but should probably display as `$0.01`.

4. **Off-ledger Amount Calculations**: It may be important for off-ledger applications to know a normative scale 
for an MPT, in order to make proper computations. Similar to the "display" example above, it will 
be important that an off-chain app knows how to interpret `100 units` of a given MPT in terms 
of a canonical unit value if, for example, adding that value to another similar value denominated 
in different units.
```

Reply:
Alright, after giving it some more thought, I decided my above suggestion is too long. I spent some time this morning to (1) shorten the overall messaging; (2) define what we mean by metadata, and (3) accurately capture the several key motivations we each have around asset scale.

I'm going to resolve this conversation for now so we can focus on the latest revision. Feel free to make suggestions or add feedback on that.